.gitignore

```diff
@@ -160,3 +160,6 @@ validation-report.json
 
 # Folders to ignore
 node_modules
+
+backward_paper/
+extras/n3lo_bench/pdf_plots
```

Comment:
let's move this down ...

Reply:
You can put in local `.gitignore` inside the folder:
```gitignore
# extras/n3lo_bench/.gitignore
pdf_plots/
```

Reply:
just to say there is already a `extras/n3lo_bench/.gitignore`

Reply:
yes sorry that was committed  unintentionally